.github/workflows/deploy.yml

```diff
@@ -13,14 +13,19 @@ jobs:
     name: Cargo Docs
     runs-on: ubuntu-latest
 
+    env:
+      RUSTDOCFLAGS: -Dbroken_intra_doc_links
```

Comment:
To avoid killing the build cache, you could use `cargo rustdoc -- -D broken_intra_doc_links`. Since you only ever run `cargo doc` once, it might not matter though.

Reply:
this does store the rust cache across runs with the swatinem/rust-cache@v1 action.  How does the env vs the cmdline arg affect the cache?

Reply:
RUSTDOCFLAGS is like RUSTFLAGS: whenever it changes, cargo assumes the whole build cache must be invalidated and effectively runs `rm -rf target/doc`. If you pass flags through `rustdoc`, it will only rebuild your crates, not your dependencies.

That said, I don't know how to make `cargo rustdoc` work with workspacers.

Reply:
That and it doesn't offer `--no-deps`, at least from what I remember. That's why we went with RUSTDOCFLAGS. It works well, since we store the build cache only between docs builds, so the flags never change between builds and the cache is not pruned.